pkg/client/client.go

```diff
@@ -139,3 +167,103 @@ func (c *Client) GetInstances(serviceType string) (map[string]string, error) {
 	}
 	return instances, nil
 }
+
+func (c *Client) CreateServiceInstance(instanceName, serviceName, servicePlan, resourceGrp, region string) (string, error) {
```

Comment:
This function needs overall more comments with descriptions and an array size checks wherever we are using [0] index